server/text_generation_server/layers/rotary.py

```diff
@@ -89,6 +89,43 @@ def static(cls, config, dim, base, device):
 
             if rope_type == "linear":
                 pass
+            elif rope_type == "longrope":
```

Comment:
Discussed in Slack: there is also another `longrope` condition later in this conditional block.

Reply:
thanks for pointing this out 🙏, i've consolidate the two longropes paths in the latest commit and only use the new class if `"short_mscale"` and `"short_mscale"` are in the config